manifests/scenario/base.pp

```diff
@@ -153,7 +153,7 @@
     }
 
     unless host_is_me($_rsync_stunnel_svr) {
-      stunnel::connection { 'rsync':
+      stunnel::instance { 'rsync':
```

Comment:
This change should be noted in the changelog. Also, is there a reason this was not done earlier? Will this have any impact on a system being upgraded?

Reply:
@lnemsick-simp Good call. I think that may have gotten in from other experimentation so I'm backing it out since it has nothing to do with this change.